lang/en/docs/software-directory/scripting/jupyter-lab/overview.md

```diff
@@ -0,0 +1,16 @@
+# Jupyter Lab
+
+
+JupyterLab [^1] is the next-generation user interface for Project Jupyter [^2] offering all the familiar building blocks of the classic Jupyter Notebook [^3] in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter Notebook.
+
+We include support for **Jupyter Lab** within our platform, using interpreter version 2.7.5 for Python programming language.
```

Comment:
1. the operating system default interpreter version 2.7.5 for Python
2. remove `programming language`

Reply:
Done.